components/Chat/ChatInput.tsx

```diff
@@ -340,6 +339,7 @@ Please remove some messages from the conversation, or simply clear all previous
             onKeyDown={handleKeyDown}
           />
           <button
+            aria-label="Send Message"
```

Comment:
Lowercase 'm': "Send messages" (we don't do 'Capital Casing' in Chatty, just 'Sentence casing').